tools/marvin/marvin/config/test_data.py

```diff
@@ -784,6 +784,18 @@
         "format": "OVA",
         "ispublic": "true"
     },
+    "coreos": {
+        "displaytext": "coreos",
+        "name": "coreos",
+        "passwordenabled": False,
+        "ostype": "Coreos",
+        "urlvmware":"http://10.147.28.7/templates/coreos/coreos_production_vmware.ova",
```

Comment:
I am now building OVA files for my templates, CoreOS is done, soon the other OSes as well.

Thanks for the help with the conversion script.